tests/integration/conftest.py

```diff
@@ -36,6 +41,66 @@ def session(request, api_url):
         yield s
 
 
+# TODO combine with session in some way?
+@pytest.fixture(scope='module',
+                params=[['admin', 'letmein']],
+                ids=['admin'])
+def girder_client(request, api_url):
+    username, password = request.param
+    client = GirderClient(apiUrl=api_url)
+    client.authenticate(username, password)
+
+    yield client
+
+
+@pytest.fixture(scope='module')
+def test_file(tmpdir_factory):
+    path = tmpdir_factory.mktemp('test').join('test.txt')
+    # Three times our read buffer for docker tests
+    path.write(''.join(random.choice(string.printable)
+                       for _ in range(1024*64*3)))
```

Comment:
If we don't care about randomness of the string, we could do `'abc'*65536` and it would be a lot faster.

Reply:
Sure, I guess I was just thinking it would give us coverage of more different chars, but I guess a byte is a byte :smile: